src/main/resources/graphics/scenery/backends/shaders/ParticleGlyphs.frag

```diff
@@ -1,25 +1,22 @@
 #version 450
 #extension GL_ARB_separate_shader_objects: enable
 
-layout(location = 0) in SilhouetteData
-{
+layout(location = 0) in SilhouetteData {
     vec3 Position;
     vec2 TexCoord;
     flat vec3 Center;
-    flat vec3 Properties;
+    flat vec2 Properties;
```

Comment:
Just checking - these are intended to be `flat` and not interpolated?

Reply:
Yes, no interpolation wanted here!